package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "tree-sitter-c",
-  "version": "0.16.0",
+  "version": "0.17.0",
```

Comment:
I'd actually like to avoid bumping the minor version number for this. At this point in Tree-sitter's development, I've been mainly using the semver version numbers to communicate binary compatibility between the Tree-sitter library and the parsers. So right now, most of the parsers are at version `0.16.x`.

Would you mind switching this to `0.16.1`?

Reply:
Done. Also adjusted on the cpp side.